### PR TITLE
[Bug]fix bug for appending hdfs url during data loading

### PR DESCRIPTION
**Pull request by QiangCai**
Reproduce:
if **FS_DEFAULT_FS**  is hdfs://bigdata.hpgdc:9000/ ,
**CARBON_DDL_BASE_HDFS_URL** is null  and 
**filePath** is /opt/spark-1.5.2-bin-hadoop2.6/carbondata/sample.csv,
method checkAndAppendHDFSUrl will return error url:
hdfs://bigdata.hpgdc:9000/null/opt/spark-1.5.2-bin-hadoop2.6/carbondata/sample.csv

Solution:
 if CARBON_DDL_BASE_HDFS_URL is null , we should not append hdfs url to file path.

### CODE REVIEW

**Comment by QiangCai** [comment]
@ravipesala please review

**Comment by Hexiaoqiao** [comment]
+1

File: integration/spark/src/test/scala/org/carbondata/spark/util/GlobalDictionaryUtilTestCase.scala [hunk 1]
```diff
@@ -46,7 +46,7 @@ class GlobalDictionaryUtilTestCase extends QueryTest with BeforeAndAfterAll {
   var complexRelation: CarbonRelation = _
   var incrementalLoadTableRelation: CarbonRelation = _
   var filePath: String = _
-  var pwd: String = _
+  var workDirectory: String = _
```
Comment (gvramana) [inline integration/spark/src/test/scala/org/carbondata/spark/util/GlobalDictionaryUtilTestCase.scala]: pwd full form is presentWorkingDirectory

**Comment by gvramana** [comment]
LGTM

**Pull request merged**